source-oracle/.snapshots/TestAllTypes-discover

```diff
@@ -269,13 +269,23 @@ Binding 0:
                     "row_id": {
                       "type": "string",
                       "description": "ROWID of the document"
+                    },
+                    "rs_id": {
+                      "type": "string",
+                      "description": "Record Set ID of the logical change"
+                    },
+                    "ssn": {
+                      "type": "integer",
+                      "description": "SQL sequence number of the logical change"
                     }
                   },
                   "type": "object",
                   "required": [
                     "schema",
                     "table",
-                    "row_id"
+                    "row_id",
+                    "rs_id",
```

Comment:
Are there any concerns about breakage to preexisting users when adding these as required? I think the answer is no (very few users at present and at least one seems to have automatic discovery disabled anyway), but just wanted to make sure this has been considered and we're confident no compatibility strategies are required.

Reply:
Yeah no real customers are using this seriously yet. Launchmetrics has been the only one who tried it but ended up not fitting their requirements and they are waiting for the batch connector